install_config/install/advanced_install.adoc

```diff
@@ -2521,6 +2521,15 @@ If for any reason the installation fails, before re-running the installer, see
 xref:installer-known-issues[Known Issues] to check for any specific
 instructions or workarounds.
 
+[CAUTION]
+====
+The installer caches playbook configuration values for 10 minutes, by default. If you change any system, network, or inventory configuration, 
+then re-run the installer within that 10 minute period, the new values are not used and the previous values are used instead. 
```

Comment:
s/used/used or s/used and the/used. The

Reply:
@kalexand-rh You are suggesting:
_...the new values are not used._ or
_...the new values are not used. The previous values are used instead. _
??

Reply:
@mburke5678, I see my typo. You either need a comma after used or to split the sentence. My apologies.